modules/images/webp-uploads/hooks.php

```diff
@@ -762,3 +762,20 @@ function webp_uploads_modify_webp_quality( $quality, $mime_type ) {
 	return $quality;
 }
 add_filter( 'wp_editor_set_quality', 'webp_uploads_modify_webp_quality', 10, 2 );
+
+/**
+ * Displays the HTML generator tag for the WebP Uploads plugin.
+ *
+ * See {@see 'wp_head'}.
+ *
+ * @since n.e.x.t
+ */
+function webp_uploads_render_generator() {
+	if (
+		defined( 'WEBP_UPLOADS_VERSION' ) &&
+		! str_starts_with( constant( 'WEBP_UPLOADS_VERSION' ), 'Performance Lab ' )
+	) {
+		echo '<meta name="generator" content="Performance Lab plugin: webp-uploads:' . WEBP_UPLOADS_VERSION . '" />' . "\n";
```

Comment:
No need to mention Performance Lab here, particularly as this will only be added in the standalone plugin. Also maybe we should use `esc_attr()` for the dynamic bit and avoid the old HTML syntax with a self-closing slash?
```suggestion
		echo '<meta name="generator" content="WebP Uploads ' . esc_attr( WEBP_UPLOADS_VERSION ) . '">' . "\n";
```